src/app/services/reducers/index.ts

```diff
@@ -12,7 +13,7 @@ import { graphExplorerMode } from './graph-explorer-mode-reducer';
 import { scopes } from './permissions-reducer';
 import { profile } from './profile-reducer';
 import { proxyUrl } from './proxy-url-reducer';
-import { sampleQuery } from './query-input-reducers';
+// import { sampleQuery } from './query-input-reducers';
```

Comment:
We can delete this

Reply:
The `query-input-reducers` and the associated test files can also be deleted since they are no longer in use